container-host-files/opt/scf/bin/klog.sh

```diff
@@ -89,6 +89,8 @@ function get_all_events() {
 rm -rf "${KLOG:?}/${NS:?}"
 NAMESPACE_DIR="${KLOG}/${NS}"
 
+mkdir -p "${NAMESPACE_DIR}"
```

Comment:
Why is this here? It seems unrelated to the topic of this PR.

Reply:
Will remove.

Reply:
Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6